### PR TITLE
Update OWNERS to remove xooglers

### PR DESCRIPTION
**Pull request by mikehelmick**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -2,14 +2,10 @@ approvers:
...
-- vaikas-google
-- n3wscott
-- mattmoor
 - nachocano
 
 # Reviewers are suggested from the reviewers list first, then the approvers
 # list. To add reviewers while spreading the load among existing approvers,
 # copy the approvers to the reviewers list too.
 reviewers:
-- n3wscott
```
Comment (nachocano) [inline OWNERS]: can you add the same set of approvers here?

Comment (mikehelmick) [reply OWNERS]: done

**Pull request merged**